### ISSUE TITLE
GoDaddy parked domains: breakage

### ISSUE THREAD

**Issue by Daxy7** [uBlockOrigin/uAssets#17532]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.ohyeah.net/`

### Category

breakage

### Description

GoDaddy parks are a blank page without allowing this filter from EasyList `||parking.godaddy.com^$3p`, which does have ads but has useful information like how to purchase the domain:
`@@||api.aws.parking.godaddy.com/v1/domains/*$xhr,3p`

Tracking and ads are blocked:
```
https://api.aws.parking.godaddy.com/d/search/p/godaddy/xml/domain/multiset/v4/?url=ohyeah.net&Partner=nameadmin_park_dm_2903_afternic&maxListings=3&maxWeb=0&maxRT=6&maxRTL=6&maxPC=6&urlLang=en&affilData=ip=10.116.88.191&xfip=10.116.88.191&ua=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/111.0&ur=http://ohyeah.net/&serveUrl=http://ohyeah.net&seedTerm=
https://api.aws.parking.godaddy.com/v1/parkingEvents?abp=1
```

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/91396482/231065844-be48b7f2-d793-4623-8a24-78a017899469.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.47.4
Firefox: 111
filterset (summary): 
  network: 79647
  cosmetic: 42767
  scriptlet: 21217
  html: 1110
listset (total-discarded, last-updated): 
  removed: 
    urlhaus-1: null
    plowe-0: null
  added: 
    https://gitlab.com/magnolia1234/bypass-paywalls-clean-filters/-/raw/5811c0757dad027282fb82f9c751c79fe4f9306e/bpc-paywall-filter.txt: 905-0, 16h.5m
    https://raw.githubusercontent.com/Daxy7/webSiteFilters/main/Ublock/getadmiral_bypass_warnings.txt: 1552-0, 16h.6m
    https://raw.githubusercontent.com/nuchi/gmail-inbox-high-contrast-restoration/master/gmail-high-contrast-restoration.txt: 61-0, 6d.9h.35m
    ublock-annoyances: 4882-4, 1d.2h.52m
  default: 
    user-filters: 1680-13, never
    ublock-filters: 34329-64, 1d.18h.53m
    ublock-badware: 6510-0, 1d.2h.53m
    ublock-privacy: 344-0, 1d.26m
    ublock-abuse: 78-0, 7h.20m
    ublock-unbreak: 1977-1, 2d.17h.40m
    ublock-quick-fixes: 352-0, 16h.7m
    easylist: 63922-262, 1d.23h.21m
    easyprivacy: 30787-2197, 1d.23h.22m
filterset (user): [array of 708 redacted]
switchRuleset: 
  added: [array of 5 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  suspendUntilListsAreLoaded: false
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 273 ms (selfie)
  maxAssetCacheWait: 330 ms
  unsuspendAfter: 103 ms
popupPanel: 
  blocked: 9
  blockedDetails: 
    ohyeah.net: 4
    godaddy.com: 4
    google.com: 1
```
</details>

**Comment by mapx-** [comment]
This way nobody will understand that site is parked.

**Comment by Daxy7** [comment]
Example: `http://www1.mailboxes.net/`

Similarly, ParkingCrew standby sites are fixed when allowing this [false positive]
```
##[data-adblockkey]:not([style$="left: -10000px !important; top: -1000px !important;"])
```
Before: White Page
After: [Domain Info](https://user-images.githubusercontent.com/91396482/232026840-cfbf1f1a-8a04-4ee8-9620-e0220352bab7.png)

**Comment by mapx-** [comment]
@Yuki2718 

maybe

```
#@#[data-adblockkey]:not([style$="left: -10000px !important; top: -1000px !important;"])
##[data-adblockkey]:not([style$="left: -10000px !important; top: -1000px !important;"]):not(html)
```

**Comment by Daxy7** [comment]
Sedo Parking is nullified aswell:

```
http://puffyforum.com/defaultsite
```

Filters: (click agree on the cookie notice)

```
@@||sedoparking.com^$3p
@@||pagead2.googlesyndication.com/apps/domainpark/*$script,3p
```

**Comment by stephenhawk8054** [comment]
`This domain name has just been registered.`

It's not parked I think?

**Comment by stephenhawk8054** [comment]
Do these work?

```adb
@@||sedoparking.com/frmpark/*/park.js|$script
@@||sedoparking.com/search/registrar.php?domain=$frame
@@||pagead2.googlesyndication.com/apps/domainpark/show_afd_ads.js|$script
||sedoparking.com/search/registrar.php$frame,removeparam=rpv,important
||sedoparking.com/search/registrar.php$frame,removeparam=gst,important
||sedoparking.com/search/registrar.php$frame,removeparam=ref,important
||sedoparking.com/search/registrar.php$inline-script
sedoparking.com##html[data-adblockkey] > body > :not(#container-header):remove()
```

**Comment by stephenhawk8054** [comment]
The original site `https://www.ohyeah.net/` displays the parked domain message fine without the need to allow `api.aws.parking.godaddy.com`, so I think we can remove it.

**Comment by Daxy7** [comment]
![image](https://github.com/uBlockOrigin/uAssets/assets/91396482/7aea5733-5c11-4cce-8556-6a41627b042f)
results

**Comment by stephenhawk8054** [comment]
> I don't really advocate for allowing sedoparking since does not specify the domain is parked. It just tries to display ads for "registered-but-not-used" domains.

**Comment by Daxy7** [comment]
maybe it will be useful for just the domain owner[s] to log into `ionos` faster? idk

**Comment by stephenhawk8054** [comment]
Domain owners can turn off uBO for that.
